### ISSUE TITLE
False negative in Style/MultilineMethodSignature

### ISSUE THREAD

**Issue by Drenmi** [rubocop-hq/rubocop#6281]
When working on the initial implementation of this cop, I had a brain fart in implementing `#definition_width`. It would take the method body into consideration when estimating the single line length of the signature, which is clearly wrong.

Thanks to @koic's review, I spotted the mistake. 🙂 

--------

## Expected behavior

The following registers an offense when `Metrics/LineLength` is set to 20:

```ruby
def foo(bar,
        baz)
  qux.quux
end
```

## Actual behavior

The inspection passes without offenses.

## Steps to reproduce the problem

Inspect the following with `Metrics/LineLength` set to 20:

```ruby
def foo(bar,
        baz)
  qux.quux
end
```

## RuboCop version

```
0.59.0 (using Parser 2.5.1.2, running on ruby 2.5.1 x86_64-darwin16)
```
